code/datums/components/heart_eater.dm

```diff
@@ -65,6 +65,8 @@
 	if(!istype(what_we_ate, /obj/item/organ/internal/heart))
 		return
 	var/obj/item/organ/internal/heart/we_ate_heart = what_we_ate
+	if (!HAS_TRAIT(we_ate_heart, TRAIT_USED_ORGAN))
```

Comment:
Preferably give some kind of balloon alert feedback here ("no life energy!"?)

Reply:
Actually `to_chat` is probably better given that this isn't blocking; they will still eat the heart.